lib/jellyfish_web/controllers/healthcheck_controller.ex

```diff
@@ -4,6 +4,8 @@ defmodule JellyfishWeb.HealthcheckController do
 
   alias JellyfishWeb.ApiSpec
 
+  @version Mix.Project.config()[:version]
+
```

Comment:
[nitpick] We're adding this module attribute in more and more places -- you could consider putting it somewhere like `jellyfish.ex`, i.e. `def version(), do: @version`, then use `Jellyfish.version()`